games/templates/play_game_as_player.html

```diff
@@ -1,66 +1,122 @@
 {% extends "base.html" %}
 
 {% block content %}
-{% if turn %}
-<h1>{{ player }} (Your turn - {{ turn.get_state_display }})</h1>
-<form method="POST" action="{% url 'turns_end_phase' turn_id=turn.pk %}">
-    {% csrf_token %}
-    <button>End Phase</button>
-</form>
-<table class="table">
-    <tbody>
-      <tr>
-        <td>Actions</td>
-        <td>{{ turn.available_actions }}</td>
-      </tr>
-      <tr>
-        <td>Buys</td>
-        <td>{{ turn.available_buys }}</td>
-      </tr>
-      <tr>
-        <td>Money</td>
-        <td>{{ turn.available_money }}</td>
-      </tr>
-    </tbody>
-</table>
-{% else %}
-<h1>{{ player }} (Not your turn)</h1>
-{% endif %}
-<h2>HAND</h2>
-<table class="table">
-    <tbody>
-      {% for card in deck.real_hand %}
-      <tr>
-        <td>{{ card }}</td>
-        <td><button>Play</button></td>
-      </tr>
-      {% endfor %}
-    </tbody>
-</table>
-<ul>
+<nav class="navbar navbar-light bg-light">
```

Comment:
Started using bootstrap. This is the only place. I might need to get ted on the horn to figure out how to make this not awful.